x-pack/filebeat/module/threatintel/_meta/docs.asciidoc

```diff
@@ -1,7 +1,7 @@
 [role="xpack"]
 
 :modulename: threatintel
-:has-dashboards: false
+:has-dashboards: true
```

Comment:
You need to run `cd filebeat; mage fmt update` to update the main docs after this change, and commit the modified files.

Reply:
Thanks @marc-gr 

Done!